documentation/pages/developer.mdx

```diff
@@ -5,6 +5,20 @@ import { Callout } from 'nextra/components'
 
 Integrate Sui Name Service (SuiNS) with your projects using either on-chain or off-chain resolutions, depending on your particular use case.
 
+## Testnet configuration
```

Comment:
Could we possibly add a new page for "Fullnode Config" and explain that:

1. On mainnet, it works by default without any config
2. On testnet (or anynet), you need to supply this in the config file as following?

My concern is that this info is for node operators and might confuse devs looking to integrate in their apps.